gtk-sys/src/lib.rs

```diff
@@ -2058,7 +2058,7 @@ pub type GtkTickCallback = Option<unsafe extern "C" fn(*mut GtkWidget, *mut gdk:
 pub type GtkTranslateFunc = Option<unsafe extern "C" fn(*const c_char, gpointer) -> *mut c_char>;
 pub type GtkTreeCellDataFunc = Option<unsafe extern "C" fn(*mut GtkTreeViewColumn, *mut GtkCellRenderer, *mut GtkTreeModel, *mut GtkTreeIter, gpointer)>;
 pub type GtkTreeDestroyCountFunc = Option<unsafe extern "C" fn(*mut GtkTreeView, *mut GtkTreePath, c_int, gpointer)>;
-pub type GtkTreeIterCompareFunc = Option<unsafe extern "C" fn(*mut GtkTreeModel, *mut GtkTreeIter, *mut GtkTreeIter, gpointer) -> c_int>;
+pub type GtkTreeIterCompareFunc = unsafe extern "C" fn(*mut GtkTreeModel, *mut GtkTreeIter, *mut GtkTreeIter, gpointer) -> c_int;
```

Comment:
It's wrapped in `Option` because Rust's `fn` is not nullable while the C function pointers are.
Now I realize, this might not be the best choice of where to put that `Option` -- it might be better at the use sites because that's where `nullable` annotations are.